README.md

```diff
@@ -459,5 +532,53 @@ bash openshift/central.sh
 </details>
 
 
+## Dependencies and Recommendations for Running StackRox
+
+<details><summary>Click to Expand</summary>
+
+The following information has been gathered to help with the installation and operation of the open source StackRox project. These recommendations were developed for the [Red Hat Advanced Cluster Security for Kubernetes](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet) product and have not been tested with the upstream StackRox project.
+
+**Tested Kubernetes Distributions**
+
+The Kubernetes Platforms that StackRox has been deployed onto with minimal issues are listed below. 
+
+- Red Hat OpenShift Dedicated (OSD)
+- Azure Red Hat OpenShift (ARO)
+- Red Hat OpenShift Service on AWS (ROSA)
+- Amazon Elastic Kubernetes Service (EKS)
+- Google Kubernetes Engine (GKE)
+- Microsoft Azure Kubernetes Service (AKS)
+
+If you deploy into a Kubernetes distribution other than the ones listed above you may encounter issues. 
+
+**Tested Operating Systems**
+
+StackRox is known to work on the recent versions of the following operating systems.
+
+- Ubuntu
+- Debian
+- Red Hat Enterprise Linux (RHEL)
+- CentOS
+- Fedora CoreOS
+- Flatcar Container Linux
+- Google COS
+- Amazon Linux
+- Garden Linux
+
+**Tested Web Browsers**
+
+The following table outlines the browsers that can view the StackRox web user interface.
+
+- Google Chrome 88.0 (64-bit)
+- Microsoft Internet Explorer Edge
+    - Version 44 and later (Windows) 
+    - Version 81 (Official build) (64-bit) (MacOS) 
+- Safari on MacOS (Mojave) - Version 14.0
```

Comment:
This is incorrect
<img width="1861" alt="image" src="https://user-images.githubusercontent.com/9895898/178548147-0fdadc80-fb59-461e-87e9-b07887d6db84.png">

Reply:
You mean the "Engineering will attempt to fix only minor issues with browsers on this tier" line should be removed since it is incorrect, right?

The tiers and browsers within the tiers look correct to me. Am I missing something?

Safari, I know, has some serious rendering issues with patternfly components. We cannot fix those issues, until patternfly support changes?

 cc: @vjwilson since we've had this browser support matrix discussion during the policies patternfly migration.

Reply:
Aah good eye there. But this is also not true and needs fixing:
"Engineering will attempt to fix only minor issues with browsers on this tier" - Policies workflow after migration to Patternfly has serious rendering issues on Safari.

Reply:
:)
I recommend removing the supports that have not been formally tested out from this readme. The downstream communication needs to be revisited.

Reply:
@Maddosaurus @md2119 Can we just say "recommended browsers"? 

We have a massive disclaimer at the top of the heading      
 
```The following information has been gathered to help with the installation and operation of the open source StackRox project. These recommendations were developed for the [Red Hat Advanced Cluster Security for Kubernetes](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet) product and have not been tested with the upstream StackRox project.```

Reply:
Works. (Not really comfortable recommending something that is not tested).